Makefile

```diff
@@ -107,3 +113,48 @@ check-dependencies-security:
 	bundle exec brakeman
+
+.PHONY: check-container-security
+check-container-security:
+	docker run --rm -v /var/run/docker.sock:/var/run/docker.sock aquasec/trivy image --no-progress --skip-dirs vendor,node_modules $(DOCKER_LOCAL_IMAGE)
+
+.PHONY: check-configuration-security
+check-configuration-security:
+	docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $$(pwd):/path aquasec/trivy config --format table /path
+#	docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $$(pwd):/path checkmarx/kics scan -p /path --no-progress --no-color -v --log-level DEBUG --exclude-paths '/path/node_modules,/path/config,/path/.github,/path/docker-compose.yml,/path/.vagrant'
+
+.PHONY: check-security
+check-security:
+	docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $$(pwd):/path aquasec/trivy fs --no-progress --format table --skip-dirs node_modules,.vagrant /path
+#	docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $$(pwd):/path anchore/grype:latest dir:/path --scope all-layers -o table
+	# docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $$(pwd):/path anchore/grype:latest sbom:/path/sbom.spdx.json -o table
+
+.PHONY: generate-sbom
+generate-sbom:
+#	docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $$(pwd):/path anchore/syft:latest packages dir:/path --scope all-layers -o spdx-json=/path/sbom.spdx.json
+#	docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $$(pwd):/path anchore/syft:latest docker:$(DOCKER_LOCAL_IMAGE) --scope all-layers -o cyclonedx-json=/path/sbom.cdx.json
+	docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $$(pwd):/path anchore/syft:latest docker:$(DOCKER_LOCAL_IMAGE) --scope all-layers -o table
+
+.PHONY: check-container-content
+check-container-content:
+#	container-structure-test test --verbosity debug --image $(DOCKER_LOCAL_IMAGE) --config cst-config.yaml
+	docker run --rm -v /var/run/docker.sock:/var/run/docker.sock gcr.io/gcp-runtimes/container-structure-test:latest test --verbosity debug --image $(DOCKER_LOCAL_IMAGE) --config cst-config.yaml
+
+.PHONY: inspect-image
+inspect-image:
+	docker run --rm -it -v /var/run/docker.sock:/var/run/docker.sock wagoodman/dive:latest $(DOCKER_LOCAL_IMAGE)
+
+.PHONY: list-container-content
+list-container-content:
+	docker create --name killswitch-container $(DOCKER_LOCAL_IMAGE)
+	docker export killswitch-container | tar t > files.txt
+
+.PHONY: sign-container
+sign-container:
+	docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $$(pwd):/path anchore/syft:latest attest --key /path/cosign.key -o cyclonedx-json $(DOCKER_LOCAL_IMAGE) > /path/sbom_att.json
+
+.PHONY: clear-security
+clear-security:
+	docker run --rm -v /var/run/docker.sock:/var/run/docker.sock aquasec/trivy image --clear-cache
+	docker run --rm -v /var/run/docker.sock:/var/run/docker.sock aquasec/trivy config --clear-cache
+	docker run --rm -v /var/run/docker.sock:/var/run/docker.sock aquasec/trivy fs --clear-cache
```

Comment:
These changes don’t seem to be related to _Upgrade to Alpine Linux 3.17 and Ruby 3.2.1._.

Reply:
My bad... I messed up but it should be fixed now.